homeassistant/components/subaru/const.py

```diff
@@ -0,0 +1,29 @@
+"""Constants for the Subaru integration."""
+
+DOMAIN = "subaru"
+DEFAULT_SCAN_INTERVAL = 300
+MIN_SCAN_INTERVAL = 60
+DEFAULT_HARD_POLL_INTERVAL = 7200
+MIN_HARD_POLL_INTERVAL = 300
+CONF_HARD_POLL_INTERVAL = "hard_poll_interval"
+
+SUPPORTED_PLATFORMS = [
+    "lock",
+    "sensor",
```

Comment:
New integrations should only have one platform in the initial PR. Probably best to start with sensor.

Reply:
Removed lock platform for this initial PR.